pymc/tests/test_distributions.py

```diff
@@ -3402,3 +3402,16 @@ def test_no_warning_logp(self):
         with pytest.warns(None) as record:
             m.logpt()
         assert not record
+
+    @pytest.mark.parametrize(
+        "sd_dist",
+        [
+            pm.Exponential.dist(1),
+            pm.MvNormal.dist(np.ones(3), np.eye(3)),
+        ],
+    )
+    def test_sd_dist_automatically_resized(self, sd_dist):
+        x = pm.LKJCholeskyCov.dist(n=3, eta=1, sd_dist=sd_dist, size=10, compute_corr=False)
+        resized_sd_dist = x.owner.inputs[-1]
+        assert resized_sd_dist.eval().shape == (10, 3)
+        assert x.eval().shape == (10, 6)
```

Comment:
I don't understand how we get to this shape from the inputs?

Reply:
The LKJCholeskyCov has support shape `(n * (n + 1)) // 2`

Reply:
OK, maybe add that as a comment.